scalalib/src/ScalaModule.scala

```diff
@@ -185,6 +201,8 @@ trait ScalaModule extends JavaModule { outer =>
       )
   }
 
+  override def docSources: Sources = T.sources(compile().classes.path)
```

Comment:
This seem not correct for early dotty / scala 3 versions (`isDotty(scalaVersion()) || isScala3Milestone(scalaVersion())`) which use dottydoc and consumes scala source files.
Only Scala 3 releases (`isScala3(scalaVersion())`) use scaladoc 3 tool, which consumes tasty files.